kodilla-patterns2/src/main/java/com/kodilla/kodillapatterns2/facade/api/OrderFacade.java

```diff
@@ -4,19 +4,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
 import org.springframework.stereotype.Service;
 
 import java.math.BigDecimal;
 
 @Service
+@EnableAspectJAutoProxy
 public class OrderFacade {
 
     @Autowired
     private ShopService shopService;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(OrderFacade.class);
 
-    public void processOrder(final OrderDto order, final Long userId) throws OrderProcessingException {
+    public void processOrder(OrderDto order, Long userId) throws OrderProcessingException {
```

Comment:
Ta zmiana nie jest potrzebna.  Wg treści modułu: `Aby dodać aspekt do klasy OrderFacade musisz usunąć słowo final z jej deklaracji` - klasy, nie metody. To klasa `OrderFacade` nie może być finalna, a nie argumenty metody `processOrder`.